deploy/olm-catalog/kogito-operator/manifests/kogito-operator.clusterserviceversion.yaml

```diff
@@ -871,5 +871,5 @@ spec:
   maturity: alpha
   provider:
     name: Red Hat
-  replaces: kogito-operator.v0.13.0
-  version: 1.0.0-snapshot
+  replaces: kogito-operator.v1.0.0-snapshot
```

Comment:
Shouldn't be here `  replaces: kogito-operator.v0.13.0`

Reply:
indeed, something difficult to handle on release pipeline side ... or we should get the different csv existing and retrieve the latest minor and/or micro version.
I guess we should correct that in the next pipeline ... https://issues.redhat.com/browse/KOGITO-3127